google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/CloudSdkStartupPolicy.java

```diff
@@ -64,15 +69,27 @@ public String getDisplayString() {
           public OSProcessHandler createProcessHandler(
               String workingDirectory, Map<String, String> envVariables) throws ExecutionException {
             CloudSdkService sdkService = CloudSdkService.getInstance();
+
             if (sdkService.getSdkHomePath() == null
-                || sdkService.getSdkHomePath().toString().isEmpty()) {
+                || StringUtils.isEmpty(sdkService.getSdkHomePath().toString())) {
```

Comment:
StringUtils#isEmpty is null safe. but I guess since its doing a toString it doesn't really matter